.travis.yml

```diff
@@ -3,7 +3,5 @@ php:
   - 5.6
   - 5.5
   - 5.4
-  - 5.3
```

Comment:
tak to muzes rovnou vyhodit aji tu 5.4 a 5.5 ne?

Reply:
:+1: bohužel 7.0+ nešlape a to teď vůbec nechci řešit

Reply:
no to bych taky neresil